src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -308,9 +307,7 @@ export default {
 		filterUrl() {
 			// initial release sends us back to /lend
 			// return `/lend/${this.$route.params.category || ''}`;
-			return this.lendFilterExpVersion === 'b'
-				? this.getAlgoliaFilterUrl()
-				: `/lend/${this.$route.params.category || ''}`;
+			return `/lend/${this.$route.params.category || ''}`;
```

Comment:
I think this one should also use `this.getAlgoliaFilterUrl()` as the return value which will check `import loanChannelQueryMapMixin from '@/plugins/loan-channel-query-map';` to get a `lend/filter` compatible url or progressively fall back to the loan channel url from the api or legacy `/lend` if needed.